code/modules/hydroponics/hydro_tray.dm

```diff
@@ -532,10 +532,8 @@
 
 		// Create a sample.
 		seed.harvest(user,yield_mod,1)
-		plant_health -= (rand(3,5)*10)
-
-		if(prob(30))
```

Comment:
i didn't manage to understand why the stage was increasing so that's the solution i came up with to avoid people being able to growth the plant using a bug......